### DISCUSSION TITLE
导入浏览器数据为空

### DISCUSSION THREAD

**Discussion by Fengxian-wang** [shion-app/shion#41] [Ideas]
### Describe the bug

使用的便携浏览器 软件导入浏览器数据失败 目录不一致导致
另外一个开发者的软件Tai使用浏览器插件解决目录不一致问题 
希望增加人工选择浏览器数据选项

### Version

v0.11.17

**Comment by Fengxian-wang** [comment]
浏览器地址栏输入
chrome://version/

可以查看数据存放地址
![1720512852151](https://github.com/shion-app/shion/assets/53722456/acd33e1b-08e4-4d9e-a26d-9bdaa1e06183)
明显和软件设置目录不一致 建议在设置里提供自由选择目录选项 当然默认目录地址可以放在建议里

**Comment by hanaTsuk1** [comment]
功能提议  我移到讨论里去的
最近是不会实现这个功能的

**Reply by Fengxian-wang** [reply]
OKOK
求一个Steam序列号
